cli/src/main/scala/io.buildo.metarpheus/cli/Cli.scala

```diff
@@ -39,23 +31,23 @@ object Cli {
       parsed <- decode[core.Config](json).toOption
     } yield parsed).getOrElse(core.Config.default)
 
-    val wiro = conf.wiro.get.getOrElse(false)
+    val wiro = conf.wiro.get.getOrElse(config.wiro)
 
-    val api = core.Metarpheus.run(sources, config.copy(wiro = wiro))
+    val api = core.Metarpheus.run(paths, config.copy(wiro = wiro))
 
     val serializedAPI = repr.serializeAPI(api)
 
-    conf.outputFile.get.map { outputFilePath =>
-      val f = new File(outputFilePath)
-      val p = new java.io.PrintWriter(f)
-      try {
-        p.println(serializedAPI)
-      } finally {
-        p.close()
-      }
+    conf.outputFile.get match {
+      case None => println(serializedAPI)
+      case Some(outputFilePath) =>
+        val f = new File(outputFilePath)
+        val p = new java.io.PrintWriter(f)
+        try {
+          p.println(serializedAPI)
+        } finally {
+          p.close()
+        }
```

Comment:
I know that this was already working, but it's not safe:

- PrintWriter doesn't throw any exception when writing, you should invoke `.checkError()`.
- `new PrintWriter(File)` doesn't automatically flush the underlying stream when writing, you should call `.flush()` manually before closing it, or `.checkError()` that implicitly does it.
- `new PrintWriter(File)` uses the underlying system's default charset. You could use `new PrintWriter(f, StandardCharsets.UTF_8.name)` to set the desired charset.